cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -177,8 +177,6 @@ def serialize_certificate(certificate):
             "name": certificate_data['name'],
             "description": certificate_data['description'],
             "version": CERTIFICATE_SCHEMA_VERSION,
-            "org_logo_path": certificate_data.get('org_logo_path', ''),
-            "is_active": certificate_data.get('is_active', False),
```

Comment:
Chris and I discussed offline -- no action necessary here -- these lines are being removed.